packages/pagination/src/pagination.js

```diff
@@ -375,27 +375,30 @@ export default {
     pageSize: {
       immediate: true,
       handler(val) {
-        this.internalPageSize = val;
+        this.internalPageSize = isNaN(val) ? 10 : val;
```

Comment:
Hi, I think it should be `1`, right?

Reply:
Hey! I though it was `10` because `pageSize`'s default value is `10` as written on prop type definitions.

https://github.com/ElemeFE/element/blob/6f4f57172b945f7f94563eb11358bb535e5639c6/packages/pagination/src/pagination.js#L14

Am I missing something? Sorry, I'm kind of new to this part.

Reply:
Yeah you're right.